### PR TITLE
New EasyConfig for GDAL 2.0.1 with the foss-2015a toolchain

### PR DESCRIPTION
**Pull request by valtandor**
Required as part of the GRASS 7.0.2 dependency tree. Requires ~~https://github.com/hpcugent/easybuild-easyconfigs/pull/2558~~

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

**Comment by boegel** [comment]
@valtandor: needs sync with `develop` as soon as #2420, #2423, #2429 are merged

File: easybuild/easyconfigs/g/GDAL/GDAL-2.0.1-foss-2015a.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ data translation and processing."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+toolchainopts = {'opt': True, 'usempi': True}
+
+source_urls = ['http://download.osgeo.org/gdal/%(version)s/']
+sources = [SOURCELOWER_TAR_XZ]
+
+dependencies = [
+    ('KEALib', '1.4.4'),
```
Comment (boegel) [inline easybuild/easyconfigs/g/GDAL/GDAL-2.0.1-foss-2015a.eb]: this dep is new, should we add it to existing GDAL easyconfigs too?

File: easybuild/easyconfigs/g/GDAL/GDAL-2.0.1-foss-2015a.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    ('netCDF', '4.3.3.1'),
+    ('HDF5', '1.8.16'),
+    ('expat', '2.1.0'),
+    ('libxml2', '2.9.3', '-Python-2.7.11'),
```
Comment (boegel) [inline easybuild/easyconfigs/g/GDAL/GDAL-2.0.1-foss-2015a.eb]: any particular reason you picked a libxml2 that was built with Python support enabled?

doesn't seem to be needed?

File: easybuild/easyconfigs/g/GDAL/GDAL-2.0.1-foss-2015a.eb (hunk 1 above)
Comment (valtandor) [inline easybuild/easyconfigs/g/GDAL/GDAL-2.0.1-foss-2015a.eb]: I believe it's only compatible with GDAL >= 2.0.0.

Comment (boegel) [inline easybuild/easyconfigs/g/GDAL/GDAL-2.0.1-foss-2015a.eb]: OK, thx

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/4899a985eefde375a36ad1b917ff8185 for a full test report.

**Comment by boegel** [comment]
lgtm

Jenkins: ok to test

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Going in, thanks @valtandor!

**Pull request merged**